driver/env/driver.go

```diff
@@ -0,0 +1,70 @@
+package env
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/buildx/driver"
+	"github.com/docker/buildx/util/progress"
+	"github.com/moby/buildkit/client"
+	buildkitclient "github.com/moby/buildkit/client"
+	"github.com/pkg/errors"
+)
+
+type Driver struct {
+	factory driver.Factory
+	driver.InitConfig
+	BuldkitdAddr string
+	BuildkitAPI  *buildkitclient.Client
+}
+
+func (d *Driver) Bootstrap(ctx context.Context, l progress.Logger) error {
+	return nil
+}
+
+func (d *Driver) Info(ctx context.Context) (*driver.Info, error) {
+	if d.BuldkitdAddr == "" && d.Driver == "env" {
+		return nil, errors.Errorf("buldkitd addr must not be empty")
+	}
+
+	c, err := client.New(ctx, d.BuldkitdAddr)
+	if err != nil {
+		return nil, errors.Wrapf(driver.ErrNotConnecting, err.Error())
```

Comment:
`fmt.Errorf` with `%w`

Reply:
`fmt.Errorf` is not allowed because it doesn't store stacktrace